packages/rxtranscribe/package.json

```diff
@@ -36,7 +36,9 @@
     "@aws-sdk/eventstream-marshaller": "^0.1.0-preview.2",
     "@aws-sdk/util-utf8-node": "^0.1.0-preview.1",
     "@bottlenose/rxws": "^0.6.0",
+    "deepspeech": "^0.6.1",
     "lodash": "^4.17.15",
+    "node-vad": "^1.1.4",
```

Comment:
This VAD standard is kind of cool.  It basically lets you ingest audio chunks and guess what is happening in the audio: silence, noise or speaking.  It can be useful for breaking the audio up into segments.